source/reference/versioning.txt

```diff
@@ -0,0 +1,119 @@
+.. _c2c-release-version-numbers:
+
+===============================
+``mongosync`` Version Numbering
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+{+c2c-product-name+} uses `Semantic Versioning 2.0.0
+<https://semver.org/>`__. Version numbers have the form ``X.Y.Z``, where
+``X`` is the major version, ``Y`` is the minor version, and ``Z`` is the
+patch version.
+
+MongoDB uses the following guidelines to determine when to increment the
+version number for {+c2c-product-name+}: 
+
+- Major number: The release breaks backwards compatibility.
+- Minor number: The release includes significant new features that are
+  backwards compatible.
+- Patch number: This release only includes small, backwards compatible
+  changes.
+
+
+Currently Supported Versions
+----------------------------
+
+If you are using an unsupported version of ``mongosync``, you may be
+asked to upgrade in order to receive support.
+
+
+MongoDB Server Version Compatibility and Support
+------------------------------------------------
+
+{+c2c-product-name+} supports the following versions of MongoDB Server:
+
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - ``mongosync`` Version
+     - MongoDB Server Version
+
+   * - 1.0.0
+     - 6.0.x
+
+Considerations
+--------------
+
+{+c2c-product-name+} has the following version related considerations:
+
+Same Server Version
+~~~~~~~~~~~~~~~~~~~
+
+{+c2c-product-name+} only supports syncing between the same version of
+MongoDB Server. All three :ref:`server version numbers
+<release-version-numbers>`, including the patch number, must be the
+same on both servers.
+
+Support Lifecycle 
+~~~~~~~~~~~~~~~~~
+
+- Major releases are supported for at least one year from the first
+  release in that series.
+- If a version of {+c2c-product-name+} only works with an unsupported 
+  version of MongoDB Server, that version of {+c2c-product-name+} is
+  also unsupported.
+
+Patch Releases
+~~~~~~~~~~~~~~
+
+Only the latest version in each major release series receives new patch
+releases. For example, when version {+c2c-product-name+} 2.1.0 is
+released, version 2.0.x would no longer receive patch releases. At the
```

Comment:
It is a bit weird to me that we say 2.0.x or 1.3.2 would receive patches. I think it would be more idiomatic to say that 2.0 or 1.3 would receive patches. i.e. just reference the minor release.